tests/IO/GeoJSONReaderTest.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace Brick\Geo\Tests\IO;
+
+use Brick\Geo\GeometryCollection;
+use Brick\Geo\IO\GeoJSONReader;
+
+class GeoJSONReaderTest extends GeoJSONAbstractTest
+{
+    /**
+     * @dataProvider providerReadGeometry
+     *
+     * @param string $geojson    The GeoJSON to read.
+     * @param array  $coords     The expected Point coordinates.
+     * @param bool   $is3D       Whether the resulting Point has a Z coordinate.
+     * @param bool   $isMeasured Whether the resulting Point has a M coordinate.
+     *
+     * @return void
+     *
+     * @throws \Brick\Geo\Exception\GeometryException
+     */
+    public function testReadGeometry(string $geojson, array $coords, bool $is3D, bool $isMeasured) : void
+    {
+        $geometry = (new GeoJSONReader())->read($geojson);
+        $this->assertGeometryContents($geometry, $coords, $is3D, $isMeasured, 4326);
+    }
+
+    /**
+     * @return \Generator
+     */
+    public function providerReadGeometry() : \Generator
+    {
+        foreach ($this->providerGeometryGeoJSON() as [$geojson, $coords, $is3D, $isMeasured]) {
+            yield [$geojson, $coords, $is3D, $isMeasured];
+            yield [$this->alter($geojson), $coords, $is3D, $isMeasured];
+        }
+    }
+
+    /**
+     * @dataProvider providerReadFeature
+     *
+     * @param string $geojson    The GeoJSON to read.
+     * @param array  $coords     The expected Point coordinates.
+     * @param bool   $is3D       Whether the resulting Point has a Z coordinate.
```

Comment:
Copy/Paste mistake: this is not a Point but a Feature. Please check other methods too, they all have the same copy/pasted doc.